applications/rubintv/values.yaml

```diff
@@ -0,0 +1,182 @@
+# Default values for rubintv.
+
+# -- Override the base name for resources
+nameOverride: ""
+
+# -- Override the full name for resources (includes the release name)
+fullnameOverride: ""
+
+# -- A special tag for letting the scripts know where they are
+# running.  Must be overridden at each site
+siteTag: ""
+
+frontend:
+  # -- If set to true, enable more verbose logging.
+  debug: false
+  # -- Prefix for rubintv's frontend API routes.
```

Comment:
Here and elsewhere, blank lines between settings before the `# --` line unless it's the first setting of a nested block, otherwise before the key of the nested block, is our formatting standard.

Reply:
I think I got that right on this second pass?  Right-ish?